client/src/Components/Mentors.js

```diff
@@ -151,8 +158,8 @@ export default function Mentors(props) {
             			New Question
 					</Link>
 				</nav><div className="container">
+					{response?(response==="OK"?<Modal setModalText={setModalText} modalText={"submitted successfully"} />:<Modal setModalText={setModalText}  modalText={"something went wrong"} />):null}
```

Comment:
so if we did get rid of the `response` state element, this could just work off `modalText` being present or not.

Reply:
exactly, thank you

Reply:
In fact not that easy, response at this moment indicates the presense of response, i need to put if statements with the actual text inside fetch error catching in this case that's the only solution that comes to my mind now